bigframes/display/anywidget.py

```diff
@@ -87,15 +91,20 @@ def __init__(self, dataframe: bigframes.dataframe.DataFrame):
         self._all_data_loaded = False
         self._batch_iter: Optional[Iterator[pd.DataFrame]] = None
         self._cached_batches: List[pd.DataFrame] = []
+        self._last_sort_state: Optional[Tuple[str, bool]] = None
```

Comment:
I can guess based on context what these mean, but I think a frozen dataclass would be much easier to understand at a glance.